### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by r7vme**
Run a daemon set that will run 5 iterations of stress with computed params.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	args := []string{
+		"--cpu", fmt.Sprintf("%v", stressCPUForks),
+		"--vm", fmt.Sprintf("%v", stressMemoryForks),
+		"--vm-hang", fmt.Sprintf("%v", stressMemoryHangSec),
+		"--timeout", fmt.Sprintf("%v", stressTimeout),
+	}
+
+	// Invoke stress command multiple times.
+	for i := 0; i < stressLoops; i++ {
+		log.Printf("Executing %s with %v", stressBinary, args)
```
Comment (r7vme) [inline main.go]: even if OOM comes for one of the stress forks Go process stays intact

File: main.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"runtime"
+)
+
+// #include <unistd.h>
```
Comment (r7vme) [inline main.go:11]: This is most convenient way to get total memory turns out

https://stackoverflow.com/questions/30021296/how-to-get-total-memory-ram-in-go

File: main.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	flag.BoolVar(&help, "help", false, "Print usage and exit")
+	flag.Parse()
+
+	// Print usage.
+	if help {
+		flag.Usage()
+		return
+	}
+
+	// Liveness probe.
```
Comment (JosephSalisbury) [inline main.go]: Why the probe?

Comment (r7vme) [reply main.go]: Good question, i thought to make sure routine can still answer. But actually no real need. So i'll remove it

File: main.go [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+		fmt.Printf("Go Version:     %s\n", runtime.Version())
+		fmt.Printf("Name:           %s\n", name)
+		fmt.Printf("OS / Arch:      %s / %s\n", runtime.GOOS, runtime.GOARCH)
+		fmt.Printf("Source:         %s\n", source)
+		return
+	}
+
+	var help bool
+
+	flag.BoolVar(&help, "help", false, "Print usage and exit")
```
Comment (JosephSalisbury) [inline main.go]: You can do something like `os.Args[1] == "--help"`, which might make this a bit tidier

**Review by JosephSalisbury — APPROVED** [review]

**Pull request merged**